internal/config/config.go

```diff
@@ -2,15 +2,19 @@ package config
 
 import (
 	"fmt"
-	"github.com/rs/zerolog"
 	"strings"
 	"time"
 
+	"github.com/rs/zerolog"
+
 	clowder "github.com/redhatinsights/app-common-go/pkg/api/v1"
 	"github.com/spf13/viper"
 )
 
 var config struct {
+	Deployment struct {
+		MetricsPort int
+	}
```

Comment:
Hmmm why deployment? All the variables are actually deployment options. I would prefer Metrics.Port here. We should try to keep the service clean of k8s/openshift stuff I think.